[2025-02-28T09:10:04.879+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T09:05:00+00:00 [queued]>
[2025-02-28T09:10:04.886+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T09:05:00+00:00 [queued]>
[2025-02-28T09:10:04.887+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-02-28T09:10:04.898+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): load_to_postgres_file> on 2025-02-28 09:05:00+00:00
[2025-02-28T09:10:04.905+0000] {standard_task_runner.py:60} INFO - Started process 6990 to run task
[2025-02-28T09:10:04.908+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl_dag', 'load_to_postgres_file', 'scheduled__2025-02-28T09:05:00+00:00', '--job-id', '146', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpu12a29in']
[2025-02-28T09:10:04.910+0000] {standard_task_runner.py:88} INFO - Job 146: Subtask load_to_postgres_file
[2025-02-28T09:10:04.956+0000] {task_command.py:423} INFO - Running <TaskInstance: crypto_etl_dag.load_to_postgres_file scheduled__2025-02-28T09:05:00+00:00 [running]> on host 335ace54b661
[2025-02-28T09:10:05.027+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl_dag' AIRFLOW_CTX_TASK_ID='load_to_postgres_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T09:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T09:05:00+00:00'
[2025-02-28T09:10:05.030+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-02-28T09:10:05.031+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/dags/scripts/load_to_postgres.py']
[2025-02-28T09:10:05.040+0000] {subprocess.py:86} INFO - Output:
[2025-02-28T09:10:05.996+0000] {subprocess.py:93} INFO - [INFO] Đã tải file '2025/02/28/16-10-02.csv' từ MinIO
[2025-02-28T09:10:05.997+0000] {subprocess.py:93} INFO - [INFO] Đã nạp 10 bản ghi vào Data Warehouse thành công.
[2025-02-28T09:10:05.998+0000] {subprocess.py:93} INFO - [INFO] Quá trình ETL hoàn tất. Đã xử lý 1 file mới từ MinIO.
[2025-02-28T09:10:06.064+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-02-28T09:10:06.086+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=crypto_etl_dag, task_id=load_to_postgres_file, execution_date=20250228T090500, start_date=20250228T091004, end_date=20250228T091006
[2025-02-28T09:10:06.124+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-02-28T09:10:06.152+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
